montecarlo/run_sim.sh

```diff
@@ -315,6 +315,7 @@ PAX_FILENAME=${PAX_INPUT_FILENAME}_pax
 HAX_FILENAME=${PAX_INPUT_FILENAME}_hax
 FAX_FILENAME=${FILENAME}_faxtruth
 LAX_FILENAME=${PAX_INPUT_FILENAME}_lax
+PAX_SR_CONFIG_FILE=${CVMFSDIRSR}/processing/montecarlo/${SCIENCERUN}_parameters
```

Comment:
Please move these .ini's into a new sub-directory.

Also, move this line below to ```FAX_PAX_CONFIG``` section.